monitor/dashboard.py

```diff
@@ -208,6 +216,14 @@ def stake_and_known_nodes_plot(n, latest_crawler_stats):
                                                    node_history=nodes_history)
             return graph
 
+        @dash_app.callback(Output('nodes-geolocation-graph', 'children'),
+                           [Input('daily-interval', 'n_intervals')],
+                           [State('cached-crawler-stats', 'children')])
+        def nodes_geographical_locations(n, latest_crawler_stats):
+            data = self.verify_cached_stats(latest_crawler_stats)
+            nodes_map = nodes_geolocation_map(nodes_dict=data['node_details'], ip2loc=self.ip2loc)
```

Comment:
Is this happening per request?
Or does it get cached for some period of time?
I would think this data only needs to be updated once every five minutes at most or something like that?

Reply:
Actually it's only updated on page load and once per day subsequently. We can shorten the interval (daily might be excessive), but it could be like per hour or something like that.